### ISSUE TITLE
Unable to deploy Single VM using bosh (concourse versions 3.9.0 and 3.9.1)

### ISSUE THREAD

**Issue by calas** [concourse/concourse#2083]
Hello, 

I'm trying to test Concourse CI using the [Single VM](http://concourse-ci.org/concourse-lite.html) solution as recommended in your documentation. But it fails to deploy concourse. Attached are the normal output and debug logs as well as the commands used: 

`BOSH_LOG_LEVEL=debug bosh create-env concourse-lite.3.9.1.yml`
[debug-output.txt](https://github.com/concourse/concourse/files/1799778/debug-output.txt)

`bosh create-env concourse-lite.3.9.1.yml --recreate`
[output.txt](https://github.com/concourse/concourse/files/1799779/output.txt)

I've also tried with version `3.9.0` and failed due to different digest hashes: 

```
$ bosh create-env concourse-lite-3.9.0.yml 
Deployment manifest: '/home/calas/dev/studio/concourse/concourse-lite-3.9.0.yml'
Deployment state: '/home/calas/dev/studio/concourse/concourse-lite-3.9.0-state.json'

Started validating
  Downloading release 'concourse'... Failed (00:00:04)
Failed validating (00:00:04)

Failed to download from 'https://github.com/concourse/concourse/releases/download/v3.9.0/concourse-3.9.0-3541.2.tgz':
  Verifying digest for downloaded file:
    Expected stream to have digest '6cc893c34ee4a504d59a06b2df3413d6e5a68e10' but was '43c8ef28fb100441efd0fb80244f979c7859f983'

Exit code 1
```

Finally I have it working with version `3.8.0`. 

My setup is: 

* **OS:** Ubuntu 16.04.4
* **Virtualbox:** 5.2.8r121009
* **bosh:** 2.0.48-e94aeeb-2018-01-09T23:08:07Z
* **Concourse:** 3.9.0 & 3.9.1

**Comment by marco-m** [comment]
@calas If your goal is to try out Concourse in a single VM, you may want to give a go at my setup for Concourse 3.9.1, based on Vagrant + VirtualBox (+ SaltStack). It is still WIP but works fine: https://github.com/marco-m/concourse-ci-formula. In case you have any problems, please open a ticket over there.

**Comment by vito** [comment]
This should be fixed with v3.9.2.

@marco-m neat!

**Comment by calas** [comment]
Confirmed working. Thanks @vito 

@marco-m Thanks for your solution too, it worked perfectly.
